e2e/scenarios/windows/property-windows-template.json

```diff
@@ -69,8 +69,8 @@
         "windowsOffer": "aks-windows",
         "windowsPublisher": "microsoft-aks",
         "windowsSku": "",
-        "windowsPauseImageURL": "mcr.microsoft.com/oss/kubernetes/pause:3.6-hotfix.20220114",
-        "alwaysPullWindowsPauseImage": "false"
+        "windowsPauseImageURL": "mcr.microsoft.com/oss/kubernetes/pause:3.9",
```

Comment:
is this related to the API changes? did tests break without this? what's the context?